ChangeLog.md

```diff
@@ -1,4 +1,12 @@
 ## Changelog
+3.2.0
+* Update node-forge and pin packages
+* Update some of the urls in README for parse
+* Add typings for the "voip" NotificationPushType
+* Typed Notification.pushType
+* Install coveralls as a dev dependency on publish
+* Fix require name in readme
+
```

Comment:
I think we should remove the GitHub / organizational PRs as they aren't relevant to devs using the changelog when updating.

Reply:
You're right. It's done.